docker-entrypoint.sh

```diff
@@ -109,6 +109,40 @@ docker_setup_env() {
     export searchd_secondary_indexes=0
   fi
 
+  if [[ -n ${CREATE_PLAIN_TABLES} && ${CREATE_PLAIN_TABLES} != "1" ]]; then
+
+    INDEXER_TABLES_LIST=""
+
+    IFS=';' read -ra ITM <<<"${CREATE_PLAIN_TABLES}"
+    for item in "${ITM[@]}"; do
+
+      IFS=':' read -ra ADDR <<<"$item"
+
+      if [ -z "${ADDR[1]}" ]; then
+        INDEXER_TABLES_LIST+=" ${ADDR[0]}"
+        continue
+      fi
+
+      if [[ ! "${ADDR[1]}" =~ ^([0-9,\-\/\*]+ )([0-9,\-\/\*]+ )([0-9,\-\/\*]+ )([0-9,\-\/\*]+ )([0-9,\-\/\*]+)$ ]]; then
+        echo -e "\033[0;31mError:\033[0m Wrong crontab syntax \033[0;31m${ADDR[1]}\033[0m for table: ${ADDR[0]}"
+        continue
+      fi
+
+      md5=$(echo -n ${ADDR[0]} | md5sum | awk '{print $1}')
+      echo "${ADDR[1]} flock -w 0 /tmp/${md5}.lock indexer --rotate ${ADDR[0]} >> /var/log/manticore/cron-${md5}.log" >> /etc/cron.d/manticore
```

Comment:
`cron-${md5}.log` may be difficult to work with as you don't know from the file name what table it is about. We need to have a table name in the log name.